_includes/header.html

```diff
@@ -1,5 +1,14 @@
 <header>
   <div class="container">
-    <p><a href="/"><img class="fadeimg header-logo" src="{{ '/assets/images/Holochain_header-logo.png' | theme_image_url }}" height="50" alt="Holochain Logo" /></a><span class="text-right no-mobile"><a href="https://github.com/Holochain/holochain-proto">GitHub</a> <a target="_blank" href="https://github.com/Holochain/holochain-proto/blob/whitepaper/holochain.pdf">Whitepaper</a> <a target="_blank" href="https://chat.holochain.org/">Chat</a>  <a href="/team.html">Team</a></span></p>
+    <p><a href="/"><img class="fadeimg header-logo" src="{{ '/assets/images/Holochain_header-logo.png' | theme_image_url }}" height="50" alt="Holochain Logo" /></a><span class="text-right no-mobile">
+      <a href="https://github.com/Holochain/holochain-proto">GitHub</a> <a target="_blank" href="https://github.com/Holochain/holochain-proto/blob/whitepaper/holochain.pdf">Whitepaper</a> <a target="_blank" href="https://chat.holochain.org/">Chat</a>  <a href="/team.html">Team</a>
+      {% if site.lang == "en" %}
+        {% capture link1 %}{{ site.baseurl_root }}es{{ page.url}}{% endcapture %}
+        <a href="{{ link1 }}" >{% t langs.es %}</a>
```

Comment:
What do you think about using flags instead of the whole word?

Also, this isn't going to work for when there's more than one language...  we're going to have to do a loop through an array of the available languages and exclude the language we are on...